### DISCUSSION TITLE
[Unified Alerting] Display Data Source Name With Rule Namespace

### DISCUSSION THREAD

**Discussion by tonychoe** [grafana/grafana#56066] [Alerting]
It would be very helpful if the Alerting UI can display the full data source name on the left of the rule namespace name. 

Especially when users need to navigate the rule list made from multiple Loki/Mimir data sources (e.g. per cloud region), the rule list doesn't quickly tell which rule belongs to which data source. If we move the mouse curser to the data source icon, the data source name appears in small text box. But this takes an action and time (a second). 

As a workaround, we started updating the rule name with the cloud region name e.g. {rule}-{region}. Not the best solution. 

![image](https://user-images.githubusercontent.com/12507518/193176864-1854a5c5-e52b-487c-83ff-5abcbc0c7e13.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
